.github/ISSUE_TEMPLATE/plugin-ownership-transfer.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Plugin Ownership Transfer
+about: Propose a plugin ownership transfer to the spinnaker-plugins organization. For more information, please see the documentation for [community donated plugins](https://github.com/spinnaker/governance/blob/master/sig-platform/community-donated-plugins.md)
+title: 'Plugin Transfer: <plugin name>'
+labels: committee/technical-oversight
+assignees: ajordens, ethanfrogers, ezimanyi, plumpy, robzienert
+
+---
+
+### Plugin Information
+
+Name of the plugin.
+
+Description of what the plugin is.
+
+### Acknowledgements
+
+- [ ] Source code will be available publicly on [`spinnaker-plugins`](https://github.com/spinnaker-plugins)
+- [ ] All authors listed below will be given `write` permissions to the plugin repository
+- [ ] Plugin authors are expected to continue as primary maintainers of plugin
+- [ ] Spinnaker core contributors are not obligated to help maintenance of plugin
```

Comment:
```suggestion
Spinnaker core contributors are not obligated to help maintain the plugin
```

Reply:
Thoughts on setting expectations even lower here, e.g. "Spinnaker core contributors are not expected to help maintain the plugin" or so?